src/chrome/content/rules/Avast.com.xml

```diff
@@ -1,44 +1,49 @@
 <!--
 	Other Avast rulesets:
-
 		- IAvs.net.xml
 
 
 	CDN buckets:
-
 		- avast.com.edgesuite.net
-
 			- sf0.iavs.net
 
 		- avast.co.jp.ldc.d3.sc.omtrdc.net
-
 			- an.avast.com
 
 
-	Nonfunctional hosts in *avast.com:
+	Redirects to http:
+		- foundation
 
-		- foundation ¹
-		- \w\w.support ¹
-		- ts ²
 
-	¹ Redirects to http
-	² 403
+	Invalid certificate:
+		- ^	( should be redirected to www. )
```

Comment:
[SSLLABS](https://www.ssllabs.com/ssltest/analyze.html?d=avast.com&s=208.43.71.133&latest) shows ok, `avast.com` is in the SAN.

Add `an.avast.com`